test/input_gen/gen_layer_tests.py

```diff
@@ -33,31 +37,57 @@
 # @brief inpsect if file is created correctly
 # @note this just checks if offset is corretly set, The result have to inspected
 # manually
-def inspect_file(file_name, _dtype = "float32"):
+
+
+def inspect_file(file_name, _dtype="float32"):
+    """_summary_
+
```

Comment:
summary contains blank lines.
However, it may not be present in other summaries.